packages/core/lib/generators/integrations/svg_integration.dart

```diff
@@ -17,11 +20,13 @@ class SvgIntegration extends Integration {
   String get classOutput => _classDefinition;
 
   String get _classDefinition => '''class SvgGenImage {
-  const SvgGenImage(this._assetName);
+  const SvgGenImage(this._assetName, {this.size = null});
 
   final String _assetName;
 ${isPackage ? "\n  static const String package = '$packageParameterLiteral';" : ''}
 
+  final Size? size;
```

Comment:
Prefer constructor before fields.

Reply:
Sorry, I don't understand the comment. The constructor is a few lines above this (just above the existing final field `_assetName`).